File: src/model_v4/entities/SBS_CRON.ts
import { Column, Entity, Index, PrimaryColumn } from "typeorm";


@Entity("SBS_CRON",{schema:"sbsdb"})
@Index("sbscron_index1",["CRON",])
export class SBS_CRON {

    @PrimaryColumn("varchar",{ 
        nullable:false,
        primary:true,
        length:20,
        name:"CRON_INDEX"
        })
    CRON_INDEX:string;
        

    @Column("varchar",{ 
        nullable:true,
        length:50,
        default: () => "'NULL'",
        name:"CRON"
        })
    CRON:string | null;
        
}

File: SbsdbServer/sbsdbv4/model/SbsCron.cs
using System.ComponentModel.DataAnnotations.Schema;

namespace hb.SbsdbServer.sbsdbv4.model {
    [Table("SBS_CRON")]
    public class SbsCron {
        [Column("CRON_INDEX", TypeName = "varchar(20)")]
        public string CronIndex { get; set; }

        [Column("CRON", TypeName = "varchar(50)")]
        public string Cron { get; set; }
    }
}
